needle/modules/dynamic/watch/traffic_capture.py

```diff
@@ -0,0 +1,139 @@
+import paramiko
+import socket
+import select
+
+from core.framework.module import BackgroundModule
+from multiprocessing import Process
+
+
+class Module(BackgroundModule):
+    meta = {
+        'name': 'Traffic Capture',
+        'author': '@Andrea Amendola',
+        'description': 'Redirect device traffic (port 80, 443) to a specific port on the workstation',
```

Comment:
The description should be a bit more verbose (i.e., explaining that is useful for apps that doesn't respect the system proxy, etc.)